docs/spec_draft.md

```diff
@@ -1586,6 +1587,59 @@ specification. Numeric precision is implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.select
+
+### Semantics
+Produces a `result` tensor where each element is selected from `on_true` or
+`on_false` tensor based on the value of the corresponding element of `pred`.
+More formally, if `predicate[i0, ..., iR-1] = true` then
+`result[i0, ..., iR-1] = on_true[i0, ..., iR-1]`  else
+`result[i0, ..., iR-1] = on_false[i0, ..., iR-1]`, where
+`predicate[i0, ..., iR-1] = pred` if `rank(pred)` $=$ `0` else
```

Comment:
This can be simplified using 

`result[i0, ..., iR-1] = predicate[i0, ..., iR-1] ? on_true[i0, ..., iR-1] : on_false[i0, ..., iR-1]` where `predicate[i0, ..., iR-1]` $=$ `rank(pred) = 0 ?  pred : pred[i0, ..., iR-1]`

Reply:
thanks